CHANGELOG.md

```diff
@@ -499,7 +499,7 @@ give, but here are some key points (from an user perspective):
   - locate now works on MLI files
   - automatic reloading of .merlin files (when they are update or created), it
     is no longer necessary to restart merlin
-  - introduced a small refactoring command: rename, who renames all occurences
+  - introduced a small refactoring command: rename, who renames all occurrences
```

Comment:
Shouldn't that "who" be a "which"?
Does the tool not check grammar, or is it to afraid to perform its magic in presence of a witch?